### PR TITLE
Refactor: Youtube embed

### PR DESCRIPTION
**Pull request by varffs**
- adds sub-object to Site.Index
- shows background image first
- hides all possible instances of youtube loader

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -367,11 +424,10 @@ Site.Doodle = {
...
-
-jQuery(document).ready(function () {
-  'use strict';
+};
 
-  Site.init();
+Site.init();
 
-});
+function onYouTubeIframeAPIReady() {
```
Comment (mikeyrayvon) [inline src/js/main.js]: is this function coming from the api library? you should specify that in a comment

File: src/js/main.js [hunk 2]
```diff
@@ -152,6 +157,58 @@ Site.Index = {
 
     $('body').stop().animate({scrollTop: shopTop}, '500', 'swing');
   },
+
+  Video: {
```
Comment (mikeyrayvon) [inline src/js/main.js:161]: can u comment this puppy?

File: src/js/main.js [hunk 3]
```diff
@@ -152,6 +157,58 @@ Site.Index = {
...
+        _this.$video.css('opacity', 1);
+
+        var duration = event.target.getDuration();
+
+        _this.timeout = setTimeout(function() {
+
+          event.target.seekTo(0);
+
+        }, (duration * 1000) - 100);
+      }
```
Comment (mikeyrayvon) [inline src/js/main.js]: elegant solution. 💯

File: templates/index.liquid [hunk 4]
```diff
@@ -5,9 +5,10 @@
 {% paginate collection.products by 12 %}
 
 {% if paginate.current_page == 1 %}
-<section id="front-splash" {% unless settings.front_background_image == blank %}style="background-image: url('{{ settings.front_background_image | img_url: '1920x' }}')"{% endunless %}>
+<section id="front-splash">
+  {% unless settings.front_background_image == blank %}<div id="front-splash-image" style="background-image: url('{{ settings.front_background_image | img_url: '1920x' }}')"{% endunless %}></div>
```
Comment (mikeyrayvon) [inline templates/index.liquid]: i think the `{% endunless %}` goes after this closing `</div>`, no?

**Review by mikeyrayvon — COMMENTED** [review]
mostly need comments in the `Video` obj. and maybe an `endunless` bug

File: src/js/main.js (hunk 2 above)
Comment (varffs) [reply src/js/main.js:161]: done

**Pull request merged**